File: app/src/main/java/app/app/astrocoin/adapters/TabAdapters.java
package app.app.astrocoin.adapters;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import app.app.astrocoin.FragmentOrder;
import app.app.astrocoin.FragmentTransfers;

public class TabAdapters extends FragmentPagerAdapter {
    private final int tabCount;

    public TabAdapters(FragmentManager fm, int tabCount) {
        super(fm);
        this.tabCount = tabCount;
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return new FragmentTransfers();
            case 1:
                return new FragmentOrder();
            default:
                return null;
        }
    }

    @Override
    public int getCount() {
        return tabCount;
    }
}

File: app/src/main/java/app/app/astrocoin/adapters/TabAdapters.kt
package app.app.astrocoin.adapters

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import app.app.astrocoin.FragmentTransfers
import app.app.astrocoin.FragmentOrder
import app.app.astrocoin.adapters.TabAdapters
import java.util.ArrayList

class TabAdapters(fm: FragmentManager) : FragmentPagerAdapter(fm) {
    override fun getItem(position: Int): Fragment {
        when (position) {
            0 -> return FragmentTransfers()
            1 -> return FragmentOrder()
        }
        return FragmentTransfers()
    }

    override fun getCount(): Int {
        return 2
    }

    fun addFragment(fragment: Fragment) {
        fragments.add(fragment)
    }

    companion object {
        private val fragments: MutableList<Fragment> = ArrayList(2)
    }
}